xml/System.Drawing/Color.xml

```diff
@@ -2163,7 +2163,7 @@
       </ReturnValue>
       <Parameters />
       <Docs>
-        <summary>Gets the hue-saturation-brightness (HSB) brightness value for this <see cref="T:System.Drawing.Color" /> structure.</summary>
+        <summary>Gets the hue-saturation-lightness (HSL) brightness value for this <see cref="T:System.Drawing.Color" /> structure.</summary>
```

Comment:
As far as I understand, Brightness and Lightness are not used interchangeably, so HSL color model does not have "brightness".  I know it sounds ridiculous "GetBrightness" returns lightness, but I think it is more accurate.

Reply:
I should have known, @Tanya-Solyanik. I couldn't find any information on brightness in the HSL model.